code/modules/paperwork/records/info/tools.dm

```diff
@@ -34,7 +34,7 @@
 	Name : Skin Prophecy<br>
 	Risk Class : Teth	<br>
 	- The employee who reads this book will experience an increase in Prudence.	<br>
-	- The book can be read until one�s Prudence reaches its limit.	<br>
+	- The book can be read until one’s Prudence reaches its limit.	<br>
```

Comment:
Is this unicode character intended?

Reply:
Turns out GitHub renders `’` weird. Using standard `'` for compatibilities sake may be better (after all, BYOND runs on like Internet Explorer Ancient Edition:tm:).

Reply:
This change is not intended, I could not get a clear answer on the proper solution prior. I'll go ahead and use the standard ' as you suggested.